load_tests/daemonset/cloudwatch.yaml

```diff
@@ -0,0 +1,248 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: load-test-fluent-bit-eks
+  namespace: load-test-fluent-bit-eks-ns
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: load-test-fluent-bit-eks-role
+rules:
+  - apiGroups: [""]
+    resources:
+      - namespaces
+      - pods
+    verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: load-test-fluent-bit-eks-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: load-test-fluent-bit-eks-role
+subjects:
+  - kind: ServiceAccount
+    name: load-test-fluent-bit-eks
+    namespace: load-test-fluent-bit-eks-ns
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloudwatch-$THROUGHPUT-config
+  namespace: load-test-fluent-bit-eks-ns
+  labels:
+    k8s-app: load-test-fluent-bit-eks
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Flush         1
+        Grace         30
+        Log_Level     info
+        Daemon        off
+        Parsers_File  parsers.conf
+        HTTP_Server   On
+        HTTP_Listen   0.0.0.0
+        HTTP_Port     2020
+        
+    @INCLUDE application-log.conf
+  
+  application-log.conf: |
+    [INPUT]
+        Name                tail
+        Tag                 kube.*
+        Path                /var/log/containers/ds-cloudwatch-$THROUGHPUT*
+        Docker_Mode         On
+        Parser              docker
+        DB                  /var/fluent-bit/cloudwatch/$THROUGHPUT/flb_log.db
```

Comment:
yeah but we have some other vars need to be replaced (not in fluent bit config) so I just use a script to replace all of them in advance: https://github.com/aws/aws-for-fluent-bit/pull/343/files#diff-3807f668d5ac75597e9a379cb6f8da0e5fb87bff143e946df9071796348e5fb5R172. rather than set env vars in fluent bit